src/components/ui/head.js

```diff
@@ -2,15 +2,21 @@ import Head from 'zefir/head'
 import React from 'react'
 
 const {OPTIMIZELY_KEY} = process.env
+const isDesktop = window.innerWidth > 768
+const robotoFontLink = <link
+    href='https://fonts.googleapis.com/css?family=Roboto:300,400,500,700'
+    rel='stylesheet' />
+const ptMonoFontLink = <link
+  href='https://fonts.googleapis.com/css?family=PT+Mono'
+  rel='stylesheet' />
 
 export default ({children}) => (
   <Head>
     <meta name='viewport' content='width=device-width, initial-scale=1' />
-    <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,500,700' rel='stylesheet' />
-    <link href='https://fonts.googleapis.com/css?family=PT+Mono' rel='stylesheet' />
     <script src={`https://cdn.optimizely.com/js/${OPTIMIZELY_KEY}.js`}></script>
     {children}
-
+    { isDesktop && ptMonoFontLink }
```

Comment:
@Idered 
I know you mentioned listeners and [this](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) but actually I want this to be triggered only on render.